src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/CSharp.xaml

```diff
@@ -50,6 +50,9 @@
     </StringProperty.Metadata>
   </StringProperty>
   <StringProperty Name="Link" Visible="false">
+    <StringProperty.DataSource>
+      <DataSource PersistenceStyle="Attribute" />
+    </StringProperty.DataSource>
     <StringProperty.Metadata>
```

Comment:
I guess `DoNotCopyAcrossProjects` is `true` by default. Does it make sense to keep`<StringProperty.Metadata> `?